tests/Commands/CreateOrEditTaskParser.test.ts

```diff
@@ -96,3 +97,47 @@ describe('CreateOrEditTaskParser - task recognition', () => {
         expect(task.doneDate).toEqualMoment(moment('2023-06-22'));
     });
 });
+
+describe('CreateOrEditTaskParser - created date', () => {
+    beforeEach(() => {
+        jest.useFakeTimers();
+        jest.setSystemTime(new Date('2023-09-17'));
+    });
+
+    afterEach(() => {
+        jest.useRealTimers();
+        resetSettings();
+    });
+
+    it.each([
+        ['- ', '- [ ]  ➕ 2023-09-17', '2023-09-17'], // bullet point only
+        ['- [ ] ', '- [ ]  ➕ 2023-09-17', '2023-09-17'], // bullet point and a checkbox
+        [
+            '- [ ] without global filter and with ➕ 2023-01-20', // with an existing created date
+            '- [ ] without global filter and with ➕ 2023-01-20',
+            '2023-01-20',
+        ],
+    ])(
+        'line loaded into "Create or edit task" command: "%s"',
+        (line: string, expectedTaskLine: string, expectedCreatedDate: string) => {
+            updateSettings({ setCreatedDate: true });
+            const path = 'a/b/c.md';
+
+            const task = taskFromLine({ line, path });
+
+            expect(task.toFileLineString()).toStrictEqual(expectedTaskLine);
+            expect(task.createdDate).toEqualMoment(moment(expectedCreatedDate));
```

Comment:
I'm curious why it tests both the line and the created date... It makes the tests a bit harder to reason about...

Reply:
True, but I wasn't sure what to pick. I can make an additional PR with just `task.createdDate`.

Reply:
OK thanks. No need to change it now... I'm gonna do some refactoring after the PR is merged, in preparation for later work, so this is fine now. Thanks.

Reply:
Thanks! =)